### DISCUSSION TITLE
Builder will replace the Website module?

### DISCUSSION THREAD

**Discussion by federicocalvo** [frappe/builder#190] [General]
My question, which I think many people have, and it is not as simple to interpret as the CRM which is obvious, is whether you are going down the road of deprecating Website in Frappe, so that Builder can be used if necessary.

The hours invested or that one must invest in the web to then do it completely differently, is what is costing to recommend, whether to go for the WebSite, or for builder.

**Comment by batonac** [comment]
My opinion: If a visual designer tool is useful to your team, go with the builder. If you have completely separate web design and development developers, you may want to retain the web/portal route, as your developer will have complete control of the code.

**Comment by federicocalvo** [comment]
I was expecting some kind of argument about the effort to maintain both modules.

Builder goes somewhere between reusable components, and templates, to perhaps a degree of simplicity like the website. I don't think it will ever get to that simplicity though.

I don't see the wisdom of keeping both. And when you start asking how this is going to fit in with Webshop(B2B) or if there is any discussion in mind about B2C, the picture is even more confusing.

**Comment by batonac** [comment]
> Builder goes somewhere between reusable components, and templates, to perhaps a degree of simplicity like the website. I don't think it will ever get to that simplicity though.

It's already pretty good once you get used to it: Jump in, the water's great!

> I don't see the wisdom of keeping both. And when you start asking how this is going to fit in with Webshop(B2B) or if there is any discussion in mind about B2C, the picture is even more confusing.

Sure, Frappe should consolidate their efforts behind builder, I agree.

You might be falling into somewhat of a false dichotomy, however. The website module isn't much more than a collection of jinja templates, and builder runs on jina under the hood, so the separation between these too isn't as dramatic as one might assume.
